oe_multilingual.install

```diff
@@ -75,6 +75,22 @@ function oe_multilingual_install() {
   _oe_multilingual_ensure_config_translations();
 }
 
+/**
+ * Implements hook_uninstall().
+ */
+function oe_multilingual_uninstall() {
+  /** @var \Drupal\oe_multilingual\LanguageNegotiationSetterInterface $setter */
+  $setter = \Drupal::service('oe_multilingual.language_negotiation_setter');
+
+  // Unset the LanguageNegotiationAdmin plugin from the interface negotiation.
+  // This is needed because the plugin is defined by this module.
+  $setter->setInterfaceSettings([
+    // By default, the URL based negotiation is the highest priority one.
+    LanguageNegotiationUrl::METHOD_ID => -19,
+    LanguageNegotiationSelected::METHOD_ID => 20,
```

Comment:
We shouldn't override the language negotiation settings with Drupal's default as the site might have changed it, we should instead load the current language negotiation settings and remove ours.